docs/unified/novacustom/firmware-update.md

```diff
@@ -31,10 +25,42 @@ firmware version is currently installed on your device.
     > binaries themselves by following the steps under the [Manual
     > update](#manual-update) section.
 
+    ### Capsule Update via fwupd
+
+    fwupd and Capsule Update are available starting with the following versions:
+
+    | Generation | Version |
+    | ---------- | ------- |
+    | 11th       | 1.6.0   |
+    | 12th       | 1.8.0   |
+    | 14th       | 1.0.0   |
+
+    !!! note
+        Make sure Intel ME is disabled before attempting Capsule Update or
+        updating via fwupd. See [Issue #1302](https://github.com/Dasharo/dasharo-issues/issues/1302)
+        for more details.
+
+    To update your firmware, run the following commands:
```

Comment:
Technically, that won't update the firmware since a reboot is necessary to apply the update.  I would mention that in some form even though `fwupd` instructs to perform a reboot, users should know that doing a shutdown will abort the update (at least with the current in-ram implementation).